mirage/dark/dark_prep.py

```diff
@@ -652,16 +652,17 @@ def linearize_dark(self, darkobj, save_refs=True):
     @logging_functions.log_fail
     def prepare(self, params=None):
         """MAIN FUNCTION"""
+        self.logger.info('\n\nRunning dark_prep..\n')
+
         # Read in the yaml parameter file
         if params is not None:
             self.params = params
         else:
             self.read_parameter_file()
 
-        # Get the log caught up on what's already happened
-        self.logger.info('\n\nRunning dark_prep..\n')
-        self.logger.info('Reading parameter file: {}\n'.format(self.paramfile))
-        self.logger.info('Original log file name: ./{}'.format(STANDARD_LOGFILE_NAME))
+            # Get the log caught up on what's already happened
+            self.logger.info('Reading parameter file: {}\n'.format(self.paramfile))
```

Comment:
If the user provides `params`, will `self.paramfile` be defined here?

Reply:
Actually, I'm realizing that I don't need to pass this dict to the dark_prep object anymore since `soss_simulator.SossSim` writes a paramfile before this is called. I'll remove the if/else from here.

Reply:
Removed!